cdap-ui/app/cdap/components/Reports/ReportsList/index.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MyReportsApi from 'api/reports';
+import Customizer from 'components/Reports/Customizer';
+import {humanReadableDate} from 'services/helpers';
+import IconSVG from 'components/IconSVG';
+import ReportsStore, {ReportsActions} from 'components/Reports/store/ReportsStore';
+import {connect} from 'react-redux';
+import Duration from 'components/Duration';
+
+require('./ReportsList.scss');
+
+class ReportsListView extends Component {
+  static propTypes = {
+    reports: PropTypes.array
+  };
+
+  componentWillMount() {
+    let params = {
+      offset: 0,
```

Comment:
nit: if this is a default should this be defaulted from the store?

Reply:
yeah, i'm putting it here for now, because eventually this will move into an action creator that will handle pagination request as well